draft-ietf-ppm-dap.md

```diff
@@ -640,6 +636,82 @@ DAP has three major interactions which need to be defined:
 Each of these interactions is defined in terms of "resources". In this section
 we define these resources and the messages used to act on them.
 
+## Message Encoding
+
+We use the presentation language defined in {{!RFC8446, Section 3}} to define
+messages in the DAP protocol, with the following deviations.
+
+{{Section 3.7 of !RFC8446}} defines a syntax for structure fields whose values
+are constants. In this document, we do not use that notation, but use something
+similar to describe specific variants of structures containing enumerated types,
```

Comment:
@tgeoghegan does this mean the syntax from `{{Section 3.7 of !RFC8446}}` is forbidden in this doc? 
If a structure has the following def:
```
struct {
   uint32 always_present;
   ExampleEnum type;
}
```

Then does the below have ambiguity?
```
struct variant {
   uint32 always_present = 0;
   ExampleEnum type = number;
}
```
Does it mean `always_present` can only be a constant 0 or it just happens to be 0 in this variant?

Reply:
It does not override that section of RFC 8446. The notation in 8446 section 3.7 is `struct { ... } whatever`. The notation we introduce here is `struct variant { ... } whatever` which does not conflict.